utils/const_error.go

```diff
@@ -0,0 +1,12 @@
+package utils
+
+// ConstError generates a const error.
+// ```
+// const SomeError = utils.ConstError("some error")
+// ```
+type ConstError string
+
+// Error returns the message of the error.
+func (e ConstError) Error() string {
```

Comment:
Why are you couldn't use errors.New("some error") from std lib `errors`?

Reply:
I hope users can't change error variables, which `const` could but `var` couldn't.

Maybe it's my overthought.

Reply:
remvoed.